<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_AddStu</class>
 <widget class="QDialog" name="Dlg_AddStu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>88</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Add New</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>87</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_name"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Age</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="sb_age">
        <property name="value">
         <number>17</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>6070</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Grade</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="le_grade"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Class</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="le_class"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>StuId</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="le_seq"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Phone</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="le_phone"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>WeiChat</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="le_weichat"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_save">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton{background-color: rgb(247, 247, 247);}

QPushButton:hover{background-color: rgb(238, 238, 238); }

QPushButton:pressed{background-color: rgb(220, 220, 220);}</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_cancle">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cancle</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_name</tabstop>
  <tabstop>sb_age</tabstop>
  <tabstop>le_grade</tabstop>
  <tabstop>le_class</tabstop>
  <tabstop>le_seq</tabstop>
  <tabstop>le_phone</tabstop>
  <tabstop>le_weichat</tabstop>
  <tabstop>btn_save</tabstop>
  <tabstop>btn_cancle</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
